projects/ngneat/error-tailor/src/lib/control-error.component.ts

```diff
@@ -20,13 +30,17 @@ import { ValidationErrors } from '@angular/forms';
     `
   ]
 })
-export class ControlErrorComponent {
+export class DefaultControlErrorComponent implements ControlErrorComponent {
   _text: string | null = null;
   _tpl: TemplateRef<{ $implicit: ValidationErrors; text: string }> | undefined;
   context: { $implicit: ValidationErrors; text: string };
   hide = true;
 
-  createTemplate(tpl: TemplateRef<any>, error: ValidationErrors, text: string) {
+  createTemplate(
+    tpl: TemplateRef<{ $implicit: ValidationErrors; text: string }>,
```

Comment:
Create a type for ` $implicit: ValidationErrors; text: string }` and use it everywhere.

Reply:
Sure.